pkg/serverless/registration/logs_api_test.go

```diff
@@ -74,7 +74,7 @@ func TestSubscribeLogsSuccess(t *testing.T) {
 		w.WriteHeader(200)
 	}))
 	defer ts.Close()
-
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Can we do anything besides waiting here? I'm not sure exactly what this tests but if it's a unit test then we should try to avoid sleeping, and maybe do something like mock it instead?

Reply:
yeah, we're mocking the http server, but it seems that for one invocation out of x, the unit test is trying to hit the http server before the initialization has been completed. I could not think about a better approach unfortunately but if you have something in mind, I'd be happy to update the PR

Reply:
All good, just checking ✅